maintaining.md

```diff
@@ -68,13 +68,13 @@ You may also want to check out the Settings page in Dev Tools and enable one or
 First collect benchmark data for a branch/commit:
 
 ```
-$ node bench/run
+$ node bench/run.js
```

Comment:
Huh, TBH I didn't try that because I just figured `node` wanted the exact file paths. Looks like it tries the various extensions instead. Neat!